src/site/scripts/components/profile/actions.tsx

```diff
@@ -37,7 +43,7 @@ export const Actions: React.StatelessComponent<IActionProps> = (props: IActionPr
 
     return (
         <div className="actions actions-alive">
-            <ActionButton action={(): void => props.onKill()} text="Report a kill!" />
-            <ActionButton action={(): void => props.onDeath()} text="Are you dead?" />
+            <ActionButton action={(): void => props.onKill()} text={killButton} confirmationText="Are you sure you want to report a kill?"/>
+            <ActionButton action={(): void => props.onDeath()} text="I'm dead!" confirmationText="Are you sure you want to report yourself as dead?"/>
```

Comment:
Nit: these lines are getting pretty wide. Can you split the attributes across lines?